# Italian translations for keystone.
# Copyright (C) 2013 ORGANIZATION
# This file is distributed under the same license as the keystone project.
#
# Translators:
msgid ""
msgstr ""
"Project-Id-Version:  Keystone\n"
"Report-Msgid-Bugs-To: https://bugs.launchpad.net/keystone\n"
"POT-Creation-Date: 2013-05-22 17:05+0000\n"
"PO-Revision-Date: 2013-05-17 16:06+0000\n"
"Last-Translator: openstackjenkins <jenkins@openstack.org>\n"
"Language-Team: Italian "
"(http://www.transifex.com/projects/p/openstack/language/it/)\n"
"Plural-Forms: nplurals=2; plural=(n != 1)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 0.9.6\n"

#: keystone/clean.py:23
#, python-format
msgid "%s cannot be empty."
msgstr "%s non può essere vuoto."

#: keystone/clean.py:25
#, python-format
msgid "%(property_name)s cannot be less than %(min_length)s characters."
msgstr "%(property_name)s non può essere inferiore a %(min_length)s caratteri."

#: keystone/clean.py:29
#, python-format
msgid "%(property_name)s should not be greater than %(max_length)s characters."
msgstr "%(property_name)s non può essere superiore a %(max_length)s caratteri."

#: keystone/clean.py:36
#, python-format
msgid "%(property_name)s is not a %(display_expected_type)s"
msgstr ""

#: keystone/test.py:115
#, python-format
msgid "Failed to checkout %s"
msgstr "Impossibile eseguire il checkout %s"

#: keystone/auth/controllers.py:72
#, python-format
msgid "Project is disabled: %s"
msgstr "Il progetto è disabilitato: %s"

#: keystone/auth/controllers.py:78 keystone/auth/plugins/password.py:40
#, python-format
msgid "Domain is disabled: %s"
msgstr "Il dominio è disabilitato: %s"

#: keystone/auth/controllers.py:84 keystone/auth/plugins/password.py:46
#, python-format
msgid "User is disabled: %s"
msgstr "L'utente è disabilitato: %s"

#: keystone/auth/controllers.py:265
msgid "Scoping to both domain and project is not allowed"
msgstr "Il controllo sia del dominio che del progetto non è consentito"

#: keystone/auth/controllers.py:268
msgid "Scoping to both domain and trust is not allowed"
msgstr "Il controllo sia del dominio che di trust non è consentito"

#: keystone/auth/controllers.py:271
msgid "Scoping to both project and trust is not allowed"
msgstr "Il controllo sia delprogetto che di trust non è consentito"

#: keystone/auth/controllers.py:333
#, python-format
msgid "Unable to lookup user %s"
msgstr "Impossibile eseguire la ricerca dell'utente %s"

#: keystone/auth/controllers.py:363
msgid "User not found"
msgstr "Utente non trovato"

#: keystone/auth/token_factory.py:81
msgid "User have no access to project"
msgstr "L'utente non ha accesso al progetto"

#: keystone/auth/token_factory.py:96
msgid "User have no access to domain"
msgstr "L'utente non ha accesso al dominio"

#: keystone/auth/token_factory.py:314 keystone/token/controllers.py:121
msgid "Unable to sign token."
msgstr "Impossibile firmare il token."

#: keystone/auth/token_factory.py:317 keystone/token/controllers.py:124
#, python-format
msgid "Invalid value for token_format: %s.  Allowed values are PKI or UUID."
msgstr ""
"Valore non valido per token_format: %s. I valori consentiti sono PKI o "
"UUID."

#: keystone/catalog/core.py:38
#, python-format
msgid "Malformed endpoint %(url)s - unknown key %(keyerror)s"
msgstr "Endpoint %(url)s non valdio - chiave sconosciuta %(keyerror)s"

#: keystone/catalog/core.py:43
#, python-format
msgid ""
"Malformed endpoint %(url)s - unknown key %(keyerror)s(are you missing "
"brackets ?)"
msgstr ""
"Endpoint %(url)s non corretto - chiave sconosciuta %(keyerror)s(mancano "
"le parentesi?)"

#: keystone/catalog/core.py:49
#, python-format
msgid ""
"Malformed endpoint %s - incomplete format                   (are you "
"missing a type notifier ?)"
msgstr ""
"Endpoint %s non corretto - formato non completo                   (manca "
"un notifier del tipo ?)"

#: keystone/catalog/backends/templated.py:109
#, python-format
msgid "Unable to open template file %s"
msgstr "Impossibile aprire il file di template %s"

#: keystone/common/bufferedhttp.py:102
#, python-format
msgid "HTTP PERF: %(time).5f seconds to %(method)s %(host)s:%(port)s %(path)s)"
msgstr "HTTP PERF: %(time).5f secondi a %(method)s %(host)s:%(port)s %(path)s)"

#: keystone/common/cms.py:44
#, python-format
msgid "Verify error: %s"
msgstr "Verifica errore: %s"

#: keystone/common/cms.py:135
msgid ""
"Signing error: Unable to load certificate - ensure you've configured PKI "
"with 'keystone-manage pki_setup'"
msgstr ""
"errore di firma: impossibile caricare il certificato - assicurarsi che "
"sia stato configurato PKI con 'keystone-manage pki_setup'"

#: keystone/common/cms.py:139
#, python-format
msgid "Signing error: %s"
msgstr "Errore di firma: %s"

#: keystone/common/config.py:93
#, python-format
msgid "Unable to locate specified logging config file: %s"
msgstr "Impossibile individuare il file config di registrazione specificato: %s"

#: keystone/common/config.py:111
msgid "Invalid syslog facility"
msgstr "Funzione syslog non valida"

#: keystone/common/controller.py:19
#, python-format
msgid "RBAC: Authorizing %s(%s)"
msgstr "RBAC: autorizzazione %s(%s)"

#: keystone/common/controller.py:27
msgid "RBAC: Invalid token"
msgstr "RBAC: token non valido"

#: keystone/common/controller.py:37 keystone/common/controller.py:58
msgid "RBAC: Invalid user"
msgstr "RBAC: utente non valido"

#: keystone/common/controller.py:43
msgid "RBAC: Proceeding without project"
msgstr "RBAC: si procede senza progetto"

#: keystone/common/controller.py:63
msgid "RBAC: Proceeding without tenant"
msgstr "RBAC: si procede senza tenant"

#: keystone/common/controller.py:93 keystone/common/controller.py:145
msgid "RBAC: Bypassing authorization"
msgstr "RBAC: autorizzazione oltrepassata"

#: keystone/common/controller.py:102 keystone/common/controller.py:143
msgid "RBAC: Authorization granted"
msgstr "RBAC: autorizzazione concessa"

#: keystone/common/controller.py:132
#, python-format
msgid "RBAC: Adding query filter params (%s)"
msgstr "RBAC: aggiunta parametri del filtro della query (%s)"

#: keystone/common/controller.py:332
msgid "Invalid token in normalize_domain_id"
msgstr "Token non valido in normalize_domain_id"

#: keystone/common/wsgi.py:106
#, python-format
msgid "Starting %(arg0)s on %(host)s:%(port)s"
msgstr "Avvio %(arg0)s in %(host)s:%(port)s"

#: keystone/common/wsgi.py:166
msgid "Server error"
msgstr "Errore del server"

#: keystone/common/wsgi.py:245
#, python-format
msgid "arg_dict: %s"
msgstr "arg_dict: %s"

#: keystone/common/wsgi.py:267
#, python-format
msgid "Authorization failed. %s from %s"
msgstr "Autorizzazione non riuscita. %s da %s"

#: keystone/common/wsgi.py:510
msgid "The resource could not be found."
msgstr "Impossibile trovare la risorsa."

#: keystone/common/ldap/core.py:79
#, python-format
msgid "Invalid LDAP deref option: %s. Choose one of: "
msgstr "Opzione deref LDAP non valida: %s. Sceglierne una di: "

#: keystone/common/ldap/core.py:87
#, python-format
msgid "Invalid LDAP tls certs option: %s. Choose one of: "
msgstr ""

#: keystone/common/ldap/core.py:96
#, python-format
msgid "Invalid LDAP scope: %s. Choose one of: "
msgstr "Ambito LDAP non valido: %s. Sceglierne uno di: "

#: keystone/common/ldap/core.py:185
#, python-format
msgid ""
"Invalid additional attribute mapping: \"%s\". Format must be "
"<ldap_attribute>:<keystone_attribute>"
msgstr ""

#: keystone/common/ldap/core.py:190
#, python-format
msgid ""
"Invalid additional attribute mapping: \"%(item)s\". Value "
"\"%(attr_map)s\" must use one of %(keys)s."
msgstr ""

#: keystone/common/ldap/core.py:274 keystone/identity/backends/kvs.py:603
#: keystone/identity/backends/kvs.py:631
#, python-format
msgid "Duplicate name, %s."
msgstr "Nome duplicato, %s."

#: keystone/common/ldap/core.py:284 keystone/identity/backends/kvs.py:596
#, python-format
msgid "Duplicate ID, %s."
msgstr "ID duplicato, %s."

#: keystone/common/ldap/core.py:289
#, python-format
msgid "LDAP %s create"
msgstr "LDAP %s crea"

#: keystone/common/ldap/core.py:367
#, python-format
msgid "LDAP %s update"
msgstr "LDAP %s aggiorna"

#: keystone/common/ldap/core.py:400
#, python-format
msgid "LDAP %s delete"
msgstr "LDAP %s elimina"

#: keystone/common/ldap/core.py:425
#, python-format
msgid "LDAP init: url=%s"
msgstr "LDAP inizializza: url=%s"

#: keystone/common/ldap/core.py:426
#, python-format
msgid ""
"LDAP init: use_tls=%(use_tls)s\n"
"tls_cacertfile=%(tls_cacertfile)s\n"
"tls_cacertdir=%(tls_cacertdir)s\n"
"tls_req_cert=%(tls_req_cert)s\n"
"tls_avail=%(tls_avail)s\n"
msgstr ""

#: keystone/common/ldap/core.py:445
msgid "Invalid TLS / LDAPS combination"
msgstr ""

#: keystone/common/ldap/core.py:449
#, python-format
msgid "Invalid LDAP TLS_AVAIL option: %s. TLSnot available"
msgstr ""

#: keystone/common/ldap/core.py:459
#, python-format
msgid "tls_cacertfile %s not found or is not a file"
msgstr ""

#: keystone/common/ldap/core.py:471
#, python-format
msgid "tls_cacertdir %s not found or is not a directory"
msgstr ""

#: keystone/common/ldap/core.py:478
#, python-format
msgid "LDAP TLS: invalid TLS_REQUIRE_CERT Option=%s"
msgstr ""

#: keystone/common/ldap/core.py:492
#, python-format
msgid "LDAP bind: dn=%s"
msgstr "LDAP bind: dn=%s"

#: keystone/common/ldap/core.py:503
#, python-format
msgid "LDAP add: dn=%s, attrs=%s"
msgstr "LDAP aggiunge: dn=%s, attrs=%s"

#: keystone/common/ldap/core.py:508
#, python-format
msgid "LDAP search: dn=%s, scope=%s, query=%s, attrs=%s"
msgstr "Ricerca LDAP: dn=%s, scope=%s, query=%s, attrs=%s"

#: keystone/common/ldap/core.py:559
msgid ""
"LDAP Server does not support paging. Disable paging in keystone.conf to "
"avoid this message."
msgstr ""
"Il server LDAP non supporta la paginazione. Disabilitare la paginazione "
"in keystone.conf per evitare questo messaggio."

#: keystone/common/ldap/core.py:576
#, python-format
msgid "LDAP modify: dn=%s, modlist=%s"
msgstr "LDAP modifica: dn=%s, modlist=%s"

#: keystone/common/ldap/core.py:581
#, python-format
msgid "LDAP delete: dn=%s"
msgstr "LDAP elimina: dn=%s"

#: keystone/common/ldap/core.py:585
#, python-format
msgid "LDAP delete_ext: dn=%s, serverctrls=%s"
msgstr "LDAP delete_ext: dn=%s, serverctrls=%s"

#: keystone/common/ldap/fakeldap.py:146
#, python-format
msgid "FakeLdap initialize url=%s"
msgstr "FakeLdap inizializza url=%s"

#: keystone/common/ldap/fakeldap.py:156
#, python-format
msgid "FakeLdap bind dn=%s"
msgstr "FakeLdap bind dn=%s"

#: keystone/common/ldap/fakeldap.py:163
#, python-format
msgid "FakeLdap bind fail: dn=%s not found"
msgstr "FakeLdap bind non riuscito: dn=%s non trovato"

#: keystone/common/ldap/fakeldap.py:170
#, python-format
msgid "FakeLdap bind fail: password for dn=%s not found"
msgstr "FakeLdap bind non riuscito: password per dn=%s non tovato"

#: keystone/common/ldap/fakeldap.py:175
#, python-format
msgid "FakeLdap bind fail: password for dn=%s does not match"
msgstr "FakeLdap bind non riuscito: password per dn=%s non corrisponde"

#: keystone/common/ldap/fakeldap.py:190
#, python-format
msgid "FakeLdap add item: dn=%s, attrs=%s"
msgstr "FakeLdap aggiunge elemento: dn=%s, attrs=%s"

#: keystone/common/ldap/fakeldap.py:192
#, python-format
msgid "FakeLdap add item failed: dn=%s is already in store."
msgstr "FakeLdap aggiunta elemento non riuscita: dn=%s è già nell'archivio."

#: keystone/common/ldap/fakeldap.py:206 keystone/common/ldap/fakeldap.py:220
#, python-format
msgid "FakeLdap delete item: dn=%s"
msgstr "FakeLdap elimina elemento: dn=%s"

#: keystone/common/ldap/fakeldap.py:210 keystone/common/ldap/fakeldap.py:224
#, python-format
msgid "FakeLdap delete item failed: dn=%s not found."
msgstr "FakeLdap elimina elemento non riuscito: dn=%s non trovato."

#: keystone/common/ldap/fakeldap.py:239
#, python-format
msgid "FakeLdap modify item: dn=%s attrs=%s"
msgstr "FakeLdap modifica elemento: dn=%s attrs=%s"

#: keystone/common/ldap/fakeldap.py:243
#, python-format
msgid "FakeLdap modify item failed: dn=%s not found."
msgstr "FakeLdap modifica elemento: dn=%s non trovato."

#: keystone/common/ldap/fakeldap.py:260
#, python-format
msgid "FakeLdap modify item failed: item has no attribute \"%s\" to delete"
msgstr ""
"FakeLdap modifica elemento non riuscito: l'elemento non ha nessun "
"attributo \"%s\" da eliminare"

#: keystone/common/ldap/fakeldap.py:271
#, python-format
msgid ""
"FakeLdap modify item failed: item has no attribute \"%s\" with value "
"\"%s\" to delete"
msgstr ""
"Modifica elemento FakeLdap non riuscita: l'elemento non ha nessun "
"attributo \"%s\" con il valore \"%s\" da eliminare"

#: keystone/common/ldap/fakeldap.py:276
#, python-format
msgid "FakeLdap modify item failed: unknown command %s"
msgstr "FakeLdap modifica elemento non riuscito: comando %s sconosciuto"

#: keystone/common/ldap/fakeldap.py:278
#, python-format
msgid "modify_s action %s not implemented"
msgstr "modify_s azione %s non implementata"

#: keystone/common/ldap/fakeldap.py:296
#, python-format
msgid "FakeLdap search at dn=%s scope=%s query=%s"
msgstr "FakeLdap ricerca in dn=%s scope=%s query=%s"

#: keystone/common/ldap/fakeldap.py:302
msgid "FakeLdap search fail: dn not found for SCOPE_BASE"
msgstr "FakeLdap ricerca non riuscita: dn non trovato per SCOPE_BASE"

#: keystone/common/ldap/fakeldap.py:316
#, python-format
msgid "Search scope %s not implemented."
msgstr "Ambito di ricerca %s non implementato."

#: keystone/common/sql/core.py:207
#, python-format
msgid "Got mysql server has gone away: %s"
msgstr "Ricevuto messaggio di interruzione della connessione del server mysql: %s"

#: keystone/common/sql/legacy.py:180
#, python-format
msgid "Cannot migrate EC2 credential: %s"
msgstr "Impossibile migrare la credenziale EC2: %s"

#: keystone/common/sql/migration.py:47
msgid "version should be an integer"
msgstr "la versione deve essere un numero intero"

#: keystone/common/sql/nova.py:62
#, python-format
msgid "Create tenant %s"
msgstr "Crea tenant %s"

#: keystone/common/sql/nova.py:79
#, python-format
msgid "Create user %s"
msgstr "Crea utente %s"

#: keystone/common/sql/nova.py:88
#, python-format
msgid "Add user %s to tenant %s"
msgstr "Aggiungi utente %s al tenant %s"

#: keystone/common/sql/nova.py:96
#, python-format
msgid "Ignoring existing role %s"
msgstr "Il ruolo esistente viene ignorato %s"

#: keystone/common/sql/nova.py:103
#, python-format
msgid "Create role %s"
msgstr "Crea ruolo %s"

#: keystone/common/sql/nova.py:113
#, python-format
msgid "Assign role %s to user %s on tenant %s"
msgstr "Assegna il ruolo %s all'utente %s nel tenant %s"

#: keystone/common/sql/nova.py:128
#, python-format
msgid "Creating ec2 cred for user %s and tenant %s"
msgstr "Creazione credenziale ec2 per l'utente %s e del tenant %s"

#: keystone/identity/backends/kvs.py:257 keystone/identity/backends/kvs.py:266
msgid "User not found in group"
msgstr "Utente non trovato nel gruppo"

#: keystone/identity/backends/sql.py:424
#, python-format
msgid "Cannot remove role that has not been granted, %s"
msgstr "Impossibile rimuovere un ruolo che non è stato concesso, %s"

#: keystone/identity/backends/ldap/core.py:95
#, python-format
msgid "Expected dict or list: %s"
msgstr ""

#: keystone/identity/backends/ldap/core.py:692
#, python-format
msgid "Role %s not found"
msgstr "Ruolo %s non trovato"

#: keystone/identity/backends/ldap/core.py:912
msgid "Changing Name not supported by LDAP"
msgstr "Modifica nome non supportato da LDAP"

#: keystone/identity/backends/ldap/core.py:926
#, python-format
msgid "User %s is already a member of group %s"
msgstr "L'utente %s è già membro del gruppo %s"

#: keystone/identity/backends/ldap/core.py:968
#, python-format
msgid ""
"Group member '%(user_dn)s' not found in '%(group_dn)s'. The user should "
"be removed from the group. The user will be ignored."
msgstr ""

#: keystone/openstack/common/policy.py:394
#, python-format
msgid "Failed to understand rule %(rule)s"
msgstr "Impossibile comprendere la regola %(rule)s"

#: keystone/openstack/common/policy.py:404
#, python-format
msgid "No handler for matches of kind %s"
msgstr "Nessun gestore per le corrispondenze di tipo %s"

#: keystone/openstack/common/policy.py:679
#, python-format
msgid "Failed to understand rule %(rule)r"
msgstr "Impossibile comprendere la regola %(rule)r"

#: keystone/policy/backends/rules.py:93
#, python-format
msgid "enforce %s: %s"
msgstr "applica %s: %s"

#: keystone/token/controllers.py:466 keystone/token/controllers.py:469
msgid "Token does not belong to specified tenant."
msgstr "Il token non appartiene al tenant specificato."

#: keystone/token/controllers.py:476
msgid "Non-default domain is not supported"
msgstr "Il dominio non predefinito non è supportato"

#: keystone/token/controllers.py:484
msgid "Domain scoped token is not supported"
msgstr "L'ambito del dominio token non è supportato"

#: keystone/token/backends/memcache.py:83
msgid "Unable to add token user list."
msgstr "Impossibile aggiungere un elenco utenti token."

#: keystone/token/backends/memcache.py:93
msgid "Unable to add token to revocation list."
msgstr "Impossibile aggiungere un token ad un elenco di revoca."

